src/main/java/uk/gov/hmcts/divorce/common/model/CaseState.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.divorce.common.model;
+
+import java.util.List;
+
+public interface CaseState {
+
+    List<String> validate(CaseData data);
```

Comment:
if you provide a default implementation you won't need to stub it for the other States